packages/components/menu/tests/menu.test.tsx

```diff
@@ -373,4 +374,81 @@ describe("<Menu />", () => {
     await act(() => fireEvent.keyDown(menuList, { key: "Escape" }))
     expect(menuList).not.toBeVisible()
   })
+  test("Context menu renders correctly", async () => {
+    const { container } = render(
+      <ContextMenu>
+        <ContextMenuTrigger
+          data-testid="ContextMenuTrigger"
+          as={Center}
+          w="full"
+          h="xs"
+          borderWidth="1px"
+          borderStyle="dashed"
+          p="md"
+          rounded="md"
+        >
+          Right click here
+        </ContextMenuTrigger>
+        <MenuList>
+          <MenuItem>Undo</MenuItem>
+          <MenuItem>Redo</MenuItem>
+        </MenuList>
+      </ContextMenu>,
+    )
+    await a11y(container)
+  })
+  test("should render the context menu", () => {
+    render(
+      <ContextMenu>
+        <ContextMenuTrigger
+          data-testid="ContextMenuTrigger"
+          as={Center}
+          w="full"
+          h="xs"
+          borderWidth="1px"
+          borderStyle="dashed"
+          p="md"
+          rounded="md"
+        >
+          Right click here
+        </ContextMenuTrigger>
+        <MenuList data-testid="MenuList">
+          <MenuItem data-testid="MenuItem">Undo</MenuItem>
+          <MenuItem>Redo</MenuItem>
+        </MenuList>
+      </ContextMenu>,
+    )
+    expect(screen.getByTestId("ContextMenuTrigger")).toBeInTheDocument()
+    expect(screen.getByText("Right click here")).toBeInTheDocument()
+    expect(screen.getByTestId("MenuList")).toBeInTheDocument()
+    expect(screen.getByTestId("MenuItem")).toBeInTheDocument()
+  })
+  test("context menu events", async () => {
+    render(
+      <ContextMenu>
+        <ContextMenuTrigger
+          data-testid="ContextMenuTrigger"
+          as={Center}
+          w="full"
+          h="xs"
+          borderWidth="1px"
+          borderStyle="dashed"
+          p="md"
+          rounded="md"
+        >
+          Right click here
+        </ContextMenuTrigger>
+        <MenuList>
+          <MenuItem>Undo</MenuItem>
+          <MenuItem>Redo</MenuItem>
+        </MenuList>
+      </ContextMenu>,
+    )
+    const contextMenuTrigger = screen.getByTestId("ContextMenuTrigger")
```

Comment:
Can the following codes be substituted?
```
const contextMenuTrigger = screen.getByText("Right click here")
```

Reply:
@108yen 
Thank you for your reviews.
I fixed it.
I have one question. Why should you avoid using getByTestId?

Reply:
@Kei-Ta 
Thank you for fixing it!
`getByTestId` is a last resort. `fireEvent` should also be used as little as possible.
Read this article.
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library